src/app/[locale]/(default)/events/[id]/layout.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@/components/ui/Button';
+import { Link } from '@/lib/locale/navigation';
+import { ArrowLeftIcon } from 'lucide-react';
+import { getTranslations } from 'next-intl/server';
+
+export default async function EventDetailsLayout({
+  children,
+}: { children: React.ReactNode }) {
+  const t = await getTranslations('events');
+  return (
+    <>
+      <Button variant='secondary' aria-label='Back to Events' asChild>
+        <Link href='/events' className='flex gap-2'>
+          <ArrowLeftIcon aria-hidden='true' />
+          <span>{t('backToEvents')}</span>
```

Comment:
It will probably work fine without a span here. Spans are mostly just needed when you need special classes on parts of an inline text